tendrl/node_agent/persistence/persister.py

```diff
@@ -62,17 +28,65 @@ def update_tendrl_context(self, context):
     def update_tendrl_definitions(self, definition):
         self._store.save(definition)
 
-    def _run(self):
-        LOG.info("Persister listening")
 
-        while not self._complete.is_set():
-            gevent.sleep(0.1)
-            pass
+class NodeAgentFilePersister(FilePersister):
```

Comment:
As discussed will remove the file persister implementation for the moment and we would add as and when required.